Sources/PipeableSDK/PipeablePage+Navigation.swift

```diff
@@ -97,6 +112,19 @@ public extension PipeablePage {
         )
     }
 
+    /// Waits for an asynchornous request made via XMLHttpRequest (XHR) or fetch to complete that matches the specified URL.
```

Comment:
Minor: but should we rename this `waitForResponse` to match Puppeteer/Playwright?

Reply:
Indeed. It's in the plans.

I'll follow up in a separate PR, since this is just docs.